Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -2372,7 +2385,7 @@ rideStatusPolling pollingId duration state push action = do
             _ <- pure $ setValueToLocalStore RIDE_STATUS_POLLING_ID (HU.generateUniqueId unit)
             _ <- pure $ setValueToLocalStore RIDE_STATUS_POLLING "False"
             if isLocalStageOn HomeScreen then pure unit
-              else doAff do liftEffect $ push $ action "CANCELLED_PRODUCT"
+              else doAff do liftEffect $ push $ action "CANCELLED_PRODUCT" HU.dummyNotificationBody
```

Comment:
you can make it maybe type instead of sending a dummy body

Reply:
Done

Reply:
No need to make it maybe, i have discuss this with prakhar